### ISSUE TITLE
Benchmark Supabase (fly.io)

### ISSUE THREAD

**Issue by kiwicopple** [supabase/benchmarks#11]
# Chore

- Running Database on Supabase
- Attached PostgREST directly to the Supabase database
- Running benchmarks directly to the PostgREST (no kong)

### Fly config

```
app = "postgrest"

[build]
  image = "postgrest/postgrest"

[[services]]
  internal_port = 3000
  protocol = "tcp"

  [services.concurrency]
    hard_limit = 25
    soft_limit = 20

  [[services.ports]]
    handlers = ["http"]
    port = "80"

  [[services.ports]]
    handlers = ["tls", "http"]
    port = "443"

  [[services.tcp_checks]]
    interval = 10000
    timeout = 2000
```

### Read benchmarks

![image](https://user-images.githubusercontent.com/10214025/99819600-9d54e580-2b8a-11eb-874b-0c0d0917e220.png)

### Write benchmarks

![image](https://user-images.githubusercontent.com/10214025/99819647-acd42e80-2b8a-11eb-9f70-94bcdf4ccc03.png)

**Comment by kiwicopple** [comment]
Just to make sure I am running with the same amount of data in the benchmark database, and on the same hardware, here is my run for Supabase:

### Read

![image](https://user-images.githubusercontent.com/10214025/99820548-e6f20000-2b8b-11eb-96b5-0dc23f620acb.png)

### Write

![image](https://user-images.githubusercontent.com/10214025/99820586-f113fe80-2b8b-11eb-881f-07b1389e96b1.png)

**Comment by kiwicopple** [comment]
#### TLDR

Setup:

- AWS: t3a.micro (Kong -> Postgres) -> database (EC2 t3a.micro)
- Fly:  (Postgres) -> database  (EC2 t3a.micro)

**Read**

- AWS: `303/s`
- Fly: `206/s`

**Write**

- AWS `307/s`
- Fly: `277/s`

### Notes

- Running from my local machine, so latency included
- Strange that read is slower than write (?)
- All systems deployed in Singapore

**Comment by awalias** [comment]
why is your AWS run so much slower than [my results](https://github.com/supabase/benchmarks/issues/8#issuecomment-729338343)

**Comment by kiwicopple** [comment]
I was wondering the same actually. I just ran [this code](https://github.com/supabase/benchmarks/tree/master/supabase) against our `Benchmarks` project. Is that how you did it?

**Comment by kiwicopple** [comment]
Benchmarks on Fly micro 1x:

![image](https://user-images.githubusercontent.com/10214025/100050092-b04e0b00-2e53-11eb-91ab-a401e8928ec3.png)

![image](https://user-images.githubusercontent.com/10214025/100050115-b8a64600-2e53-11eb-81b0-13e5543b953c.png)

**Comment by kiwicopple** [comment]
I'll add a comparison table to the top comment
